tests/integration/docker_test/drivers.dockerfile

```diff
@@ -1,37 +1,30 @@
 FROM debian:stable-slim
```

Comment:
whole file has beed changed from merge conflicts, it needs to be reverted to what was on the left